Tests/SonarRunner.Shim.Tests/SonarProjectPropertiesValidatorTest.cs

```diff
@@ -0,0 +1,122 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarQube.Common;
+using System;
+using System.IO;
+using System.Collections.Generic;
+
+namespace SonarRunner.Shim.Tests
+{
+    #region Tests
+
+    [TestClass]
+    public class SonarProjectPropertiesValidatorTest
+    {
+        public TestContext TestContext { get; set; }
+
+        [TestMethod]
+        public void SonarProjectPropertiesValidatorTest_FailCurrentDirectory()
+        {
+            var folder = Path.Combine(TestContext.TestRunResultsDirectory, "FailCurrentDirectory");
+            Directory.CreateDirectory(folder);
+            File.Create(Path.Combine(folder, "sonar-project.properties"));
```

Comment:
FYI there are a couple of helper methods in TestUtils that will create test-specific folders and files e.g. TestUtils.CreateTestSpecificFolder(this.TestContext)